### PR TITLE
CHAT-810:Problem with meeting notes when there's a task

### PR DESCRIPTION
**Pull request by bourasom**
What is wrong is that in the ReportBean there is a mapping issue with message object sent from task's chat integration side. The assignees are got by username and set in the username option not in fullname's(It does not exist)

### CODE REVIEW

**Comment by boubaker** [comment]
Could you please describe the fix and what was causing the bug

File: server-embedded/src/main/java/org/exoplatform/chat/model/ReportBean.java [hunk 1]
```diff
@@ -142,14 +142,14 @@ else if ("type-hand".equals(options.get("type").toString()))
...
-            task.setAssignee(options.get("fullname").toString());
+            task.setAssignee(fullname);
             task.setTask(options.get("task").toString());
             task.setDueDate(options.get("dueDate").toString());
             task.setUsername(options.get("username").toString());
             addTask(task);
             msg = "[ "+ res.getString("exoplatform.chat.meetingnotes.task").
               replace("{0}", options.get("task").toString()).
-              replace("{1}", options.get("fullname").toString()).
+              replace("{1}", options.get("username").toString()).
```
Comment (boubaker) [inline server-embedded/src/main/java/org/exoplatform/chat/model/ReportBean.java]: why should we display the username instead the fullname ?

Comment (bourasom) [reply server-embedded/src/main/java/org/exoplatform/chat/model/ReportBean.java]: The full name given is task creator's not assignees.

Comment (bourasom) [reply server-embedded/src/main/java/org/exoplatform/chat/model/ReportBean.java]: "Could you please describe the fix and what was causing the bug"
What is wrong is that in the ReportBean there is a mapping issue with message object sent from task's chat integration side. The assignees are got by username and set in the username option not in fullname's(It does not exist)

Comment (boubaker) [reply server-embedded/src/main/java/org/exoplatform/chat/model/ReportBean.java]: ok, so we will display the username in message too.

File: server-embedded/src/main/java/org/exoplatform/chat/model/ReportBean.java [hunk 2]
```diff
@@ -142,14 +142,14 @@ else if ("type-hand".equals(options.get("type").toString()))
           else if ("type-task".equals(options.get("type").toString()))
           {
             TaskBean task = new TaskBean();
-            task.setAssignee(options.get("fullname").toString());
+            task.setAssignee(fullname);
```
Comment (boubaker) [inline server-embedded/src/main/java/org/exoplatform/chat/model/ReportBean.java]: This should be equal to `options.get("username").toString()`
In fact, fullname is equal to current username who has saved the wiki page.

File: server-embedded/src/main/java/org/exoplatform/chat/model/ReportBean.java [hunk 3]
```diff
@@ -142,14 +142,14 @@ else if ("type-hand".equals(options.get("type").toString()))
...
             task.setTask(options.get("task").toString());
             task.setDueDate(options.get("dueDate").toString());
```
Comment (boubaker) [inline server-embedded/src/main/java/org/exoplatform/chat/model/ReportBean.java]: The wiki page is not saved if assignee or due date are empty

**Pull request closed without merging**